tests/semantic_analyser.cpp

```diff
@@ -181,6 +182,12 @@ TEST(semantic_analyser, call_hist)
   test("kprobe:f { hist(); }", 1);
 }
 
+TEST(semantic_analyser, call_lhist)
+{
+  test("kprobe:f { @ = lhist(5, 0, 10, 1); }", 0);
+  test("kprobe:f { @ = lhist(-10, -10, 10, 1); }", 10); // must be positive
```

Comment:
It would also make sense to check that lhist only works when assigned to a map:
`test("kprobe:f { lhist(5, 0, 10, 1); }", 1);`

Reply:
Looks like this causes a segfault. Filed #760 for it. Let's fix it in another PR.